test-app/tests/integration/components/headless-form-basic-test.gts

```diff
@@ -154,537 +142,4 @@ module('Integration Component HeadlessForm > Basics', function (hooks) {
-        .hasAttribute('type', 'radio')
-        .hasAttribute('name', 'choice')
-        .hasValue('foo')
-        .hasClass('my-input', 'it accepts custom HTML classes')
-        .hasAttribute(
-          'data-test-radio',
-          '',
-          'it accepts arbitrary HTML attributes'
-        );
-    });
-
-    test('label and input are connected', async function (this: RenderingTestContext, assert) {
-      const data: { choice?: string } = {};
-
-      await render(<template>
-        <HeadlessForm @data={{data}} as |form|>
-          <form.Field @name="choice" as |field|>
-            <field.Radio @value="foo" as |radio|>
-              <radio.Input />
-              <radio.Label>Foo</radio.Label>
-            </field.Radio>
-          </form.Field>
-        </HeadlessForm>
-      </template>);
-
-      assert.dom('input').hasAttribute(
-        'id',
-        // copied from https://ihateregex.io/expr/uuid/
-        /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/,
-        'input has id with dynamically generated uuid'
-      );
-
-      const id = this.element.querySelector('input')?.id ?? '';
-
-      assert
-        .dom('label')
-        .hasAttribute(
-          'for',
-          id,
-          'label is attached to input by `for` attribute'
-        );
-    });
-
-    test('checked property is mapped correctly to @data', async function (assert) {
-      const data: { choice?: string } = { choice: 'bar' };
-
-      await render(<template>
-        <HeadlessForm @data={{data}} as |form|>
-          <form.Field @name="choice" as |field|>
-            <field.Radio @value="foo" as |radio|>
-              <radio.Input data-test-radio1 />
-              <radio.Label>Foo</radio.Label>
-            </field.Radio>
-            <field.Radio @value="bar" as |radio|>
-              <radio.Input data-test-radio2 />
-              <radio.Label>Bar</radio.Label>
-            </field.Radio>
-          </form.Field>
-        </HeadlessForm>
-      </template>);
-
-      assert.dom('input[data-test-radio1]').isNotChecked();
-      assert.dom('input[data-test-radio2]').isChecked();
-    });
-  });
-
-  module('field.Textarea', function () {
-    test('field yields textarea component', async function (assert) {
-      const data: { comment?: string } = {};
-
-      await render(<template>
-        <HeadlessForm @data={{data}} as |form|>
-          <form.Field @name="comment" as |field|>
-            <field.Textarea class="my-textarea" data-test-textarea />
-          </form.Field>
-        </HeadlessForm>
-      </template>);
-
-      assert
-        .dom('textarea')
-        .exists('render a textarea')
-        .hasAttribute('name', 'comment')
-        .hasClass('my-textarea', 'it accepts custom HTML classes')
-        .hasAttribute(
-          'data-test-textarea',
-          '',
-          'it accepts arbitrary HTML attributes'
-        );
-    });
-  });
-
-  module('data', function () {
```

Comment:
this whole suite was already extracted to `headless-form-data-test.gts`, but somehow still left here. So this is removing the duplication.